lib/array_intersection.rb

```diff
@@ -1,6 +1,26 @@
+require "pry"
 # Returns a new array to that contains elements in the intersection of the two input arrays
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: Time complexity is O(n) (or, O(n+m)) we are looping through one array O(n), then searching worst case for a look up a hash table O(n)
+# Space complexity: O(n) with n elements in the hash table.
```

Comment:
I take it `n` in the time and space complexities are the same? (It never hurts to be explicit when discussing complexity.)